docs/03-apis/api-admin/permissions.md

```diff
@@ -19,12 +19,119 @@ License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
 
 # Permissions Endpoint
 
-  - **Add/change/delete administrative permissions**:
+##Permission Operations:
 
-      - Required permission: SystemAdmin / hasProjectAllAdminPermission
-        / hasProjectRightsAdminPermission
+**Note:** For the following operations, the requesting user must be either a `systemAdmin`, 
+a `projectAdmin` or a `systemUser`.
```

Comment:
As a side note, access as `systemUser` shouldn't be allowed through the API. The `systemUser` should only be used internally and with care, so that information is not leaked.

Reply:
From your explanation of systemUser, I gather that only `or a systemUser` must be deleted here.

Reply:
Yes